src/automata_pccs/shared/AutomataDaoStorage.sol

```diff
@@ -25,25 +53,90 @@ contract AutomataDaoStorage is Ownable {
     }
 
     function revokeDao(address revoked) external onlyOwner {
-        _authorized[revoked] = false;
+        _authorized_writers[revoked] = false;
+    }
+
+    function collateralPointer(bytes32 key) external pure override returns (bytes32 collateralAttId) {
+        collateralAttId = key;
     }
 
-    function writeToPccs(bytes32 attId, bytes memory attData) external onlyAuthorized(msg.sender) {
-        _pccsData[attId] = attData;
+    function collateralHashPointer(bytes32 key) external pure override returns (bytes32 collateralHashAttId) {
+        collateralHashAttId = bytes32(uint256(key) + 1);
     }
 
-    function deleteData(bytes32 attId) external onlyAuthorized(msg.sender) {
-        delete _pccsData[attId];
+    function readAttestation(bytes32 attestationId)
+        external
+        view
+        override
+        onlyDao(msg.sender)
+        returns (bytes memory attData)
+    {
+        attData = _db[attestationId];
     }
 
-    function readPccs(bytes32 attId) external view returns (bytes memory attData) {
-        attData = _pccsData[attId];
+    /**
+     * @notice In AutomataDaoStorage, we will simply assign the key as the attestationid of the collateral
+     * @notice whereas the value (key + 1) will be the attestation id to the hash.
```

Comment:
add a prefix to the pre-image to differentiate collateral and hash attestations

Reply:
resolved [5d7dcbe](https://github.com/automata-network/automata-on-chain-pccs/pull/9/commits/5d7dcbe67437f94ff7d4603b727ca8e2764b65e6)